MIGRATIONS.md

```diff
@@ -3,6 +3,12 @@
 This file documents changes in the data model. Please explain any changes to the
 data model as well as any custom migrations.
 
+## WordPress 93
+
+@guarani 2019-10-27
+
+- `BasePost` added `autosaveTitle` (`nullable` `String`), `autosaveExcerpt` (`nullable` `String`), `autosaveContent` (`nullable` `String`), and `autosaveModifiedDate` (`nullable` `Date`) properties.
```

Comment:
Thanks for catching that - I had originally put them in `BasePost`, then moved them to `AbstractPost`. 

--------

The WordPress [support article](https://wordpress.org/support/article/revisions/#autosaves) talks about "autosaves" (no hyphen) - however, consensus appears divided.
- Favors hyphen: https://english.stackexchange.com/a/7659/164338
- Favors **_no_** hyphen: https://www.collinsdictionary.com/spellcheck/english?q=autosave (no results for hyphenated version)

I have no strong opinion either way - let me know if you think it would be better capitalized and I'd be happy to oblige.

Reply:
I think you're right. Let's keep it at `"autosave"`.